### PR TITLE
Separate CLI and application

### PR DESCRIPTION
**Pull request by philipp94831**

### CODE REVIEW

File: streams-bootstrap/src/main/java/com/bakdata/kafka/util/ImprovedAdminClient.java [hunk 1]
```diff
@@ -28,62 +28,121 @@
...
+            try {
+                this.schemaRegistryClient.close();
+            } catch (final IOException e) {
+                throw new UncheckedIOException("Error closing schema registry client", e);
+            }
+        }
+    }
+
+    @RequiredArgsConstructor
+    private static class PooledAdmin implements Admin {
```
Comment (philipp94831) [inline streams-bootstrap/src/main/java/com/bakdata/kafka/util/ImprovedAdminClient.java:123]: Move to separate PR

**Review by raminqaf — APPROVED** [review]
The migration worked flawlessly.

**Review by JakobEdding — APPROVED** [review]
